themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -968,6 +968,7 @@ ldap.connection-pooling.prefsize.tooltip=The string representation of an integer
 ldap.connection-pooling.protocol.tooltip=A list of space-separated protocol types of connections that may be pooled. Valid types are "plain" and "ssl".
 ldap.connection-pooling.timeout.tooltip=The string representation of an integer that represents the number of milliseconds that an idle connection may remain in the pool without being closed and removed from the pool.
 ldap.pagination.tooltip=Does the LDAP server support pagination.
+ldap.tls.tooltip=Does the LDAP server require STARTTLS.
```

Comment:
Is it possible to improve the tooltip and add also the information that enabling "Use STARTTLS" will also effectively disable connection pooling? Also is it possible to add the validation check to the LDAPStorageProviderFactory.validateConfiguration method and throw an exception if both "Use StartTLS" and connection pooling is enabled? This will make sure that it's more "hard" to wrongly configure things.

Reply:
Hi @mposolda, `LDAPConfig.java` won't allow them to both be enabled at the same time so this should never occur https://github.com/keycloak/keycloak/pull/6038/files#diff-2cb2e46d543ba030c2e8ec7b958ef125L114

I will update the validateConfiguration shortly anyways.